spec/acceptance/firewall_attributes_exceptions_spec.rb

```diff
@@ -19,9 +19,8 @@
 
   # --bytecode is only supported by operatingsystems using nftables (in general Linux kernel 3.13, RedHat 7 (and derivates) with 3.10)
   # Skipping those from which we know they would fail.
-  describe 'bytecode property', unless: (os[:family] == 'redhat' && os[:release][0] <= '6') ||
```

Comment:
we should be also good to remove :
```
(os[:family] == 'sles' && os[:release][0..1] <= '11')
```
as we don't have 11, we only support 12 & 15.

Reply:
Done.